src/core/public/application/integration_tests/application_service.test.tsx

```diff
@@ -106,9 +106,13 @@ describe('ApplicationService', () => {
           },
         });
 
-        const { navigateToApp, currentAppId$ } = await service.start(startDeps);
+        const { navigateToApp, currentAppId$, getComponent } = await service.start(startDeps);
+        update = createRenderer(getComponent());
 
-        await act(() => navigateToApp('app1'));
+        await act(async () => {
+          await navigateToApp('app1');
+          update();
+        });
```

Comment:
emission is now only fired when the app is mounted, so had to adapt this test to actually mount the router as we do in some other tests of the file